BootloaderCorePkg/Tools/PrepareBuildComponentBin.py

```diff
@@ -246,7 +257,15 @@ def Main():
         CopyBins (fsp_repo_dir, sbl_dir, fsp_inf)
 
     microcode_inf = os.path.join(sbl_dir, 'Silicon', silicon_pkg_name, 'Microcode', 'Microcode.inf')
-    CopyBins (ucode_repo_dir, sbl_dir, microcode_inf)
+    microcode_bins = CopyBins (ucode_repo_dir, sbl_dir, microcode_inf)
+
+    # Add microcode padding
+    if microcode_bins != None and len(sys.argv) > 4:
+        slot_size = int(sys.argv)
```

Comment:
Looks its reusing the argument. Add a new parameter here.

Reply:
Should now be fixed.